opus/application/static_media/js/browse.js

```diff
@@ -280,7 +280,7 @@ var o_browse = {
 
         // Disable draggable for these infomation modals
         // If draggable is enabled, once modal is dragged, it will not stay in the center.
-        for (let id of ["#op-browser-version-msg", "#op-browser-size-msg", "#op-guide"]) {
+        for (let id of ["#op-browser-version-msg", "#op-browser-size-msg", "#op-new-user-msg"]) {
```

Comment:
didn't notice this before; probably should have a data item instead of updating this list that is something like "data-draggable=true"

Reply:
Done